Controllers/MeetingsController.cs

```diff
@@ -0,0 +1,61 @@
+using LinkUpBackend.DTOs;
+using LinkUpBackend.Models;
+using LinkUpBackend.Infrastructure;
+using LinkUpBackend.Domain;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity;
+
+
+namespace Meetings.Controllers;
+[ApiController]
+[Route("api/[controller]")]
+public class MeetingsController : Controller{
+    private readonly AppDbContext dbContext;
+    private readonly UserManager<User> userManager;
+    public MeetingsController(AppDbContext dbContext, UserManager<User> userManager){
+        this.dbContext = dbContext;
+        this.userManager = userManager;
+    }
+    [HttpGet]
+    public async Task<IActionResult> GetMeetings(){
+        return Ok(await dbContext.Meetings.ToListAsync());
+    }
+    
+    [HttpPost]
+    public async Task<IActionResult> AddMeeting(AddMeetingRequestDTO request){
```

Comment:
Maybe add [From Body] to parameter as you shall get request body with data to map?